verification_tests/1.SrVO3_uspp/sirius.json

```diff
@@ -1,13 +1,20 @@
-{
+{ 
+
+"control" : {
     "cyclic_block_size" : 16,
     "processing_unit" : "cpu",
     "std_evp_solver_type" : "lapack",
-    "gen_evp_solver_type" : "lapack",
-    "num_fv_states" : 40,
+    "gen_evp_solver_type" : "lapack"
+},
+
+
+"parameters" : {
+ 
     "electronic_structure_method" : "ultrasoft_pseudopotential",
     
     "xc_functionals" : ["XC_LDA_X", "XC_LDA_C_PZ"],
 
+    "!num_fv_states" : 40,
     "iterative_solver" : {
```

Comment:
"iterative_solver" is not part of parameters, but a separate section. Please recommit the changes